test/kpod.bats

```diff
@@ -47,3 +51,17 @@ KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
 	echo "$output"
 	[ "$status" -ne 0 ]
 }
+
+@test "kpod inspect image" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS pull redis:alpine
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS inspect redis:alpine
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
```

Comment:
Give me a test with the format command looking for a specific field.
Give me a test with --type container or --type image.